opentera-webrtc-native-client/OpenteraWebrtcNativeClient/python/package/setup.py.in

```diff
@@ -24,18 +28,17 @@ setup(
     url="https://github.com/introlab/opentera-webrtc",
     packages=['opentera_webrtc_native_client'],
     classifiers=[
-       "Programming Language :: Python :: 3",
-       "Development Status :: 4 - Beta",
-       "License :: OSI Approved :: Apache Software License",
-       "Operating System :: POSIX",
+        "Programming Language :: Python :: 3",
+        "Development Status :: 4 - Beta",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: POSIX",
     ],
+    license="Apache Software License",
     distclass=BinaryDistribution,
     platforms=['linux_x86_64'],
     # python_requires='==3.8',
     package_dir={'opentera_webrtc_native_client': '.'},
-    package_data={'opentera_webrtc_native_client': ['opentera_webrtc_native_client.cpython-38-x86_64-linux-gnu.so']},
+    package_data={'opentera_webrtc_native_client': ["$<TARGET_FILE_NAME:${PYTHON_LIB_NAME}>"]},
+    $<IF:$<BOOL:${OPENTERA_WEBRTC_ENABLE_PYTHON_HTML_DOC}>,data_files=[('doc'$<COMMA> ['opentera_webrtc_native_client.html'])]$<COMMA>,>
     include_package_data=True,
-    data_files=[('doc', ['opentera_webrtc_native_client.html'])],
```

Comment:
Dynamically get the .so file name, and only include html doc if `BUILD_HTML_DOC` is `ON`